modules/docs/src/main/scala/weaver/Output.scala

```diff
@@ -35,3 +35,79 @@ object Output {
     } yield value.reduceLeft(_ + "\n" + _)
   }
 }
+
+
+import fansi.Str
+
+object Ansi2Html extends Function1[String, String] {
```

Comment:
This is not a bulletproof implementation - most of it was just to make weaver's examples work.

It probably misses a ton of corner cases.

I can fix it if it ever becomes a problem - ideally I'd want to extract it as a separate mini-project.